index.html

```diff
@@ -4,471 +4,16 @@
     <!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
+    <meta http-equiv="refresh" content="3;url=https://m2d2.io/" />
```

Comment:
the number `3` here is simply the number of seconds after which the redirect will happen. 

decrease or increase that number to make the redirect faster/slower